lib/dynamics_crm/response/result.rb

```diff
@@ -11,7 +11,8 @@ def initialize(xml)
         @document = REXML::Document.new(xml)
 
         fault_xml = @document.get_elements("//[local-name() = 'Fault']")
-        raise XML::Fault.new(fault_xml) if fault_xml.any?
+        code = fault_xml.first.get_text("//[local-name() = 'Code']/[local-name() = 'Value']")  if fault_xml.any?
```

Comment:
This exception was raised even in case of successful response for 'ExecuteMultiple' request, due to:

`
(...)
<d:ExecuteMultipleResponseItem>
                <d:Fault i:nil='true'/>
(...)
`